diesel/src/backend.rs

```diff
@@ -19,17 +33,43 @@ where
     Self: HasSqlType<types::Time>,
     Self: HasSqlType<types::Timestamp>,
 {
+    /// The concrete `QueryBuilder` implementation for this backend.
```

Comment:
Link to `QueryBuilder`

Reply:
This renders underneath a link to `QueryBuilder`.